src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/dft.cpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <single_layer_tests/dft.hpp>
+#include <common_test_utils/test_constants.hpp>
+
+namespace {
+
+const std::initializer_list<ngraph::helpers::DFTOpType> opTypes {
+    ngraph::helpers::DFTOpType::FORWARD,
+//    ngraph::helpers::DFTOpType::INVERSE
```

Comment:
is it expected to be commented?

Reply:
it will be implemented as part of next pr for idft

Reply:
could you either remove it and add in next PR or add comments about this with TODO - as now it looks little bit confusing

Reply:
added comment

Reply:
implemented idft